accelerator/apis.c

```diff
@@ -9,7 +9,21 @@
 #include "apis.h"
 #include "map/mode.h"
 
-static logger_id_t logger_id;
+#define APIS_LOGGER_ID "apis"
```

Comment:
How about `APIS_LOGID` or `APIS_LOG_ID`?

Reply:
This is my negligence. I think the name of the constant should be APIS_LOGGER(without ID) since it doesn't contain any meaning of ID.